octopusdeploy/resource_account.go

```diff
@@ -48,13 +55,17 @@ func resourceAccount() *schema.Resource {
 			},
 			"tenanted_deployment_participation": {
 				Type:     schema.TypeString,
-				Required: true,
+				Optional: true,
```

Comment:
Technically this is not optional, it is likely filled in by the API. We should default this to the right value like we do with other such values to be consistent